### ISSUE TITLE
Module not found: Can't resolve './QRCode' in '/work/projects/test-projects/reactjs-minimal-app/src'

### ISSUE THREAD

**Issue by siarhei-kudrautsau** [authenteq/reactjs-minimal-app#1]
**Hi Gays!**
I try build your app
And in next step I have it error
**Module not found: Can't resolve './QRCode' in '<my local path...>/reactjs-minimal-app/src'**

_I use node of 8.10 before run app, my npm proposaled to me fix broken version of installed packajes
what I have now_

```bash
--- a/package.json
+++ b/package.json
@@ -10,8 +10,8 @@
     "express": "^4.16.3",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
-    "react-scripts": "1.1.4",
-    "sockjs-client": "^1.1.4",
+    "react-scripts": "^2.0.4",
+    "sockjs-client": "^1.3.0",
     "stompjs": "^2.3.3"
   },
   "scripts": {
@@ -21,5 +21,11 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
-  }
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all"
+  ]
 }
```
and
------
```bash
--- a/src/App.js
+++ b/src/App.js
@@ -3,7 +3,7 @@ import * as api from './api';
 import logo from './logo.png';
 import './App.css';
 import Form from './Form';
-import QRCode from './QRCode';
+//import QRCode from './QRCode';
```
------
You see I commented it out " import QRCode.." becose it folder not exists in your repository

you wrote
Both QR and AQR code scanning is supported in Android app from v1.24.13, in iOS
app from v1.20.0. 
Support for QR code scanning will be removed in future
version of Android and iOS app.

**Pleas comment me, how it work ?**

**Comment by pavel-lens** [comment]
Hi Sergey, let me check it and correct it. Thanks for reporting.

**Comment by pavel-lens** [comment]
Resolved in 6f38c43. I added few more checks to make sure you put your `AUTHENTEQ_PARTNER_ID` and `AUTHENTEQ_API_KEY` into `.env` file.

You can checkout master branch.
